tfjs-backend-webgpu/src/kernels/concat_webgpu.ts

```diff
@@ -28,10 +28,12 @@ export class ConcatProgram implements WebGPUProgram {
   dispatchLayout: {x: number[]};
   dispatch: [number, number, number];
   variableNames: string[];
+  uniforms = '';
   workPerThread = 4;
   workGroupSize: [number, number, number] = [64, 1, 1];
   shapes: Array<[number, number]>;
   size = true;
+  offsetLength: number;
 
   constructor(shapes: Array<[number, number]>) {
```

Comment:
It seems that you only need a `offsetsLength`. How about using `constructor(offsetsLength: number)` and removing `shapes`?

Reply:
Shape is used for this.outputShape

Reply:
Yes, I missed that.